index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+	<header>
+		<a href="#" class="header-text">
+			<h3>
+				Welcome
+			</h3>
+		</a>
+		<nav>
+			<img src="./img/background/hamberger.svg" alt="menu">
+		</nav>
+	</header>
+	<section class="headline">
+		<h1>
+			Hey there. I’m Yoseph
+		</h1>
+		<h1>
+			<span class="header_green">I’m a software developer</span>
+		</h1>
```

Comment:
- [ ] Why don't you use one h1 tag for this to make your code clean like this :point_down: 
  ```html
    <h1>
	    Hey there. I’m Yoseph
	    <span class="header_green">I’m a software developer</span>
    </h1>
  ```